fegzycole/React-Calculator
# Complete Milestone One Of The React Calculator App

### What Does This PR Do?
- Completes all requirements for milestone 1 which include
  - Create application using create-react-app
  - Delete unnecessary files
  - Install big.js and prop-types npm packages
  - Set up project linters
  - Deploy App on Heroku

### How Can It Be Tested?
- To see the application hosted on heroku, click [This](https://fegzycole-calculator.herokuapp.com/) link
